Lib/test/support/__init__.py

```diff
@@ -2118,3 +2119,19 @@ def clear_ignored_deprecations(*tokens: object) -> None:
     if warnings.filters != new_filters:
         warnings.filters[:] = new_filters
         warnings._filters_mutated()
+
+
+# This number should be CPython internal "hot" count times 2.
```

Comment:
I assume that the "hot" count is QUICKENING_WARMUP_DELAY, if so you can you add a comment pointing to that macro for the count ?

Reply:
yeap, thanks.